crates/egui/src/style.rs

```diff
@@ -1052,18 +1055,17 @@ impl Visuals {
         self.window_stroke
     }
 
-    /// When fading out things, we fade the colors towards this.
-    // TODO(emilk): replace with an alpha
+    /// When fading out things, we multiply the opacity by this.
     #[inline(always)]
-    pub fn fade_out_to_color(&self) -> Color32 {
-        self.widgets.noninteractive.weak_bg_fill
+    pub fn fade_out_opacity(&self) -> f32 {
```

Comment:
Let's use the same name everywhere

```suggestion
    pub fn disabled_alpha(&self) -> f32 {
```